treeherder/webapp/api/serializers.py

```diff
@@ -400,6 +400,27 @@ def validate_tree(self, tree):
         return tree
 
 
+class HashQuerySerializer(serializers.Serializer):
+    basehash = serializers.CharField()
```

Comment:
Florin suggested using serializers.IntegerField() instead of serializers.CharField(). This might allow you to remove the `validate_basehash` and `validate_newhash` methods